content/en/docs/about/community.md

```diff
@@ -5,30 +5,22 @@ weight = 10
 aliases = ["/docs/community/"]
 +++
 
-## Kubeflow Slack
-
-The Official Kubeflow Slack is used for informal discussions among users and contributors.
-
-<a href="https://communityinviter.com/apps/kubeflow/slack">
-  <button class="btn btn-primary py-2 px-5 mb-3">Click to join:<br>Kubeflow Slack</button>
-</a>
-
-| Popular Kubeflow Slack Channels |
-| --- | --- |
-| General Discussion | [#general](https://kubeflow.slack.com/archives/C7REE0ETX)
-| Feature Requests | [#feature-requests](https://kubeflow.slack.com/archives/C01A7RYEYMB)
-| Job Postings | [#job-postings](https://kubeflow.slack.com/archives/CJ9PJE5FS)
-| Kubeflow - Pipelines | [#kubeflow-pipelines](https://kubeflow.slack.com/archives/CE10KS9M4)
-| Kubeflow - Notebooks | [#kubeflow-notebooks](https://kubeflow.slack.com/archives/CESP7FCQ7)
-| Kubeflow - Training Operator | [#kubeflow-training-operator](https://kubeflow.slack.com/archives/C985VJN9F)
-| Kubeflow - Katib | [#kubeflow-katib](https://kubeflow.slack.com/archives/CE0BURK1B)
-| KServe | [#kserve](https://kubeflow.slack.com/archives/CH6E58LNP)
-| Platform - AWS | [#platform-aws](https://kubeflow.slack.com/archives/CKBA5D0MU)
-| Platform - Azure | [#platform-azure](https://kubeflow.slack.com/archives/CUW6SLCPR)
-| Platform - GCP | [#platform-gcp](https://kubeflow.slack.com/archives/CKH7V1M7F)
-| Users - China | [#users-china](https://kubeflow.slack.com/archives/C93HYNM9C)
-| Users - Korea | [#users-korea](https://kubeflow.slack.com/archives/CKPCJB9AP)
-| Users - Oceania | [#users-oceania](https://kubeflow.slack.com/archives/C023ZN1R9FC)
+## Kubeflow Slack Channels
+
+Kubeflow is part of the vibrant CNCF community. You are welcome to create your CNCF Slack account [here](https://slack.cncf.io/) and then join the following list of Kubeflow Slack channels.
```

Comment:
@terrytangyuan @andreyvelich I see the above conversation (https://github.com/kubeflow/website/pull/3739/files#r1610529091) was marked as resolved without reason.

So here is my suggestion again:

```suggestion
Kubeflow is part of the vibrant CNCF community, we use the [CNCF Slack](https://slack.cncf.io/) for informal discussions among users and contributors.
Please join the [Kubeflow channels](#slack-channels) to join the conversation and get help from the community.

<a href="https://slack.cncf.io/">
  <button class="btn btn-primary py-2 px-5 mb-3">Click to join:<br><b>CNCF Slack</b></button>
</a>
```

Reply:
That's because I marked it as resolved based on your previous comment https://github.com/kubeflow/website/pull/3739/files#r1610472753 and then you added another comment on the resolved comment.

Reply:
But you marked both as resolved?

Either way, can we please discuss keeping the button, and the slight rewording proposed above?